modules/ROOT/pages/configuring-mule-stacktraces.adoc

```diff
@@ -1,69 +1,71 @@
-= Configuring Mule Stacktraces
+= Configuring Mule Stack Traces
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: anypoint, studio, mule, on premises, on premise, amc, logs, stacktraces, verbose
+:keywords: anypoint, studio, mule, on premises, on premise, amc, logs, stack traces, verbose
 
-By default Mule does not show stacktraces and even filters out some internal class
- references from stacktraces to  produce a more readable output when it does show them.
-This behavior can be controlled in one of two ways:
+By default, Mule does not display all stack traces. When Mule does show stack traces, it filters out some internal class references from to produce a more readable output.
```

Comment:
`it filters out some internal class references from to produce a more readable output` this looks wrong, what does the `from` mean?

Reply:
Fixed